aten/src/ATen/native/quantized/cpu/qnnpack/CMakeLists.txt

```diff
@@ -429,7 +430,7 @@ if(PYTORCH_QNNPACK_BUILD_TESTS)
     CXX_STANDARD_REQUIRED YES
     CXX_EXTENSIONS NO)
   target_include_directories(deconvolution-test PRIVATE src test)
-  target_link_libraries(deconvolution-test PRIVATE pytorch_qnnpack cpuinfo gtest gtest_main)
+  target_link_libraries(deconvolution-test PRIVATE pytorch_qnnpack cpuinfo fp16 gtest gtest_main)
```

Comment:
Why do we need fp16 library for this?

Reply:
FP16 is required for the QNNPACK requantization